docs/projects/sovereign-boot-wizard.md

```diff
@@ -21,42 +21,68 @@ Wizard
 Specification](https://github.com/3mdeb/verified-boot/blob/55df31bf0645767e18c4dac7e19bf62d788e9df8/verified_boot_app_spec.md#sovereign-boot-provisioning-wizard)
 (current revision v0.1.0).
 
+## Credits
+
+This research has been supported by [Power Up
+Privacy](https://powerupprivacy.com/), a privacy advocacy group that seeks to
+supercharge privacy projects with resources so they can complete their mission
+of making our world a better place.
+
+Author of the idea and motivation: [Patrick Schleizer](https://github.com/adrelanos).
+
+Design and technical support: [Aaron Rainbolt](https://github.com/ArrayBolt3).
```

Comment:
@ArrayBolt3 let me know if you would like to change this line

Reply:
I'm fine with that, and thanks for mentioning me :)